includes/data-port/models/class-sensei-import-course-model.php

```diff
@@ -110,6 +110,19 @@ public function sync_post() {
 		$this->set_course_terms( Sensei_Data_Port_Course_Schema::COLUMN_MODULES, 'module', $teacher );
 		$this->set_course_terms( Sensei_Data_Port_Course_Schema::COLUMN_CATEGORIES, 'course-category' );
 
+		/**
+		 * Associations task object for this job.
+		 *
+		 * @var Sensei_Import_Associations $associations_task
+		 */
+		$associations_task = $this->task->get_job()->get_associations_task();
+		$associations_task->add_course_lessons(
```

Comment:
Maybe skip this if no lessons are provided? Or is the column mandatory?

Reply:
Skipped if `null` but still want it to run if empty 997583dcf24e510d663bab82761b5f11e0754e03